pytorch_pfn_extras/training/extensions/_snapshot.py

```diff
@@ -367,6 +367,7 @@ def initialize(self, manager):
                         target[k].load_state_dict(state[k], **kwargs)
                 else:
                     target.load_state_dict(state, **kwargs)
+                writer.fs.close(snapshot_file)
```

Comment:
In PFIO, this is an interface to shutdown the filesystem itself. We should call close on file instead.

```suggestion
                snapshot_file.close()
```

Note: HdfsFile object provides `close` method
https://arrow.apache.org/docs/python/generated/pyarrow.HdfsFile.html#pyarrow.HdfsFile